pubnub-kotlin/pubnub-kotlin-api/src/iosMain/kotlin/com/pubnub/api/PubNubImpl.kt

```diff
@@ -818,8 +818,7 @@ class PubNubImpl(private val pubNubObjC: KMPPubNub) : PubNub {
     }
 
     override fun getToken(): String? {
-        // TODO:
-        return null
+        return pubNubObjC.configObjC().authToken()
```

Comment:
so in swift when someone calls `pubnub.setToken` it is saved in `pubnub.config.authToken` ?

Reply:
I agreed with Marcin that I'll change this getter to method. Te final invocation should look like:

```
return pubNubObjC.configObjC().getToken()
```